ipaserver/install/adtrustinstance.py

```diff
@@ -838,12 +847,19 @@ def find_local_id_range(self):
         )
         api.Backend.ldap2.add_entry(entry)
 
+    def __retrieve_local_range(self):
+        """Retrieves local IPA ID range to make sure
+        """
+        self.sub_dict['IPA_LOCAL_RANGE'] = get_idmap_range(self.realm)
+
+
```

Comment:
Too many empty lines

Reply:
`make fastcheck` did not highlight any of these errors on F30

Reply:
I think the pep8 checker would have complained. Thanks!